cf-serverd/server_common.c

```diff
@@ -373,8 +373,7 @@ static void AbortTransfer(ConnectionInfo *connection, char *filename)
     Log(LOG_LEVEL_VERBOSE, "Aborting transfer of file due to source changes");
 
     char sendbuffer[CF_BUFSIZE];
-    snprintf(sendbuffer, CF_BUFSIZE, "%s%s: %s",
-             CF_CHANGEDSTR1, CF_CHANGEDSTR2, filename);
+    snprintf(sendbuffer, CF_BUFSIZE, "%s: %s", CF_CHANGEDSTR, filename);
```

Comment:
Why are you changing / cleaning up cf-serverd code? I thought we agreed to implement the new protocol functions, independently of the old functionality. That way we can ensure that we don't fix / change any parts of the implementation, and test cf-net with an "unaltered" cf-serverd. Also, why is there a diff in the commit message?

Reply:
We did agree to that, but at the time of making these changes, I was not aware of `CF_CHANGEDSTR` actually having an effect on functionality. The other changes I made are just code cleanup, and I want to split those into a new PR now.

The diff is there by accident, as you might have guessed. Something probably went wrong when I wanted to reuse the commit message in `COMMIT_EDITMSG` when a commit failed.